appi.adoc

```diff
@@ -290,49 +290,100 @@ corresponding to a different field construct.
+points are connected as well). For a given coordinate construct, all
+the parts of all the cells must be of the same one of these three
+kinds, which are called "geometry types". The bounds array spans the
+domain axis constructs of the coordinate construct, with the addition
+of two trailing ragged dimensions. The first extra dimension indexes
+the parts of each cell and the second indexes the points that describe
+each part.
+
+If cell bounds are provided for a dimension coordinate construct then
+each cell must have exactly two vertices forming a line geometry. For
+climatological time coordinates the actual cell extent comprises
+multiple time segments equivalent to multiple line geometry parts, but
+the bounds require just two points to define each cell, namely the
+earliest and latest times of the sequence. The cell method constructs
+indicate how the multiple time segments should be inferred from these
+climatological bounds.
+
+If a cell is composed of multiple polygon parts, an individual polygon
+may define an "interior ring", i.e. a region that is to be omitted
+from, as opposed to included in, the cell extent. In this case an
+interior ring array is required that records whether each polygon is
+to be included or excluded from the cell, and is supplied by an
+interior ring variable in CF-netCDF. The interior ring array spans the
+domain axis constructs of the coordinate construct, with the addition
+of an extra ragged dimension that indexes the parts for each cell. For
+example, a cell describing the land area surrounding a lake would
+require two polygon parts: one defines the outer boundary of the land
+area; the other, recorded as an interior ring, is the lake boundary,
+defining the inner boundary of the land area.
+
 If a domain axis construct does not correspond to a continuous
 physical quantity, then it is not necessary for it to be associated
 with a dimension coordinate construct. For example, this is the case
 for an axis that runs over ocean basins or area types, or for a domain
-axis that indexes a time series at scattered points. In such cases,
-one-dimensional auxiliary coordinate constructs could be used to store
-coordinate values. These axes are called "discrete" axes in CF-netCDF.
+axis that indexes a time series at scattered points. These axes are
+discrete axes in CF-netCDF. In such cases cells may be described with
+one-dimensional auxiliary coordinate constructs for which, provided
+that there is a cell bounds array to describe the cell extents, the
+coordinate array is optional, since coordinates are not always well
+defined for such cells. A CF-netCDF geometry container variable is
+used to store cell bounds without coordinates for a discrete axis.
+
+In CF-netCDF, when a CF-netCDF geometry container variable is present,
+it explicitly describes the geometry type and the number of parts per
+cell for the coordinate variables that it references. In this case, a
```

Comment:
The geometry container variable indicates which variables contain the number of parts per cell -- it doesn't do it its self.

Reply:
Good catch, thanks. Text updated.